frontend/src/app/modules/intervention-form/file-uploader/file-uploader.component.ts

```diff
@@ -48,8 +47,9 @@ export class FileUploaderComponent implements OnInit {
     );
   }
 
-  onFileRemove(removedFile: File) {
+  onFileRemove(removedFile: File, input: HTMLInputElement) {
     this.files = this.files.filter(file => file !== removedFile);
+    input.value = '';
```

Comment:
It seems that resetting the input was needed and had an effect here ;) Input value needs to be reset on file remove, so that the file can be re-added if user decides to add it again :)